src/test/java/io/redisearch/client/ClientJsonTest.java

```diff
@@ -0,0 +1,161 @@
+package io.redisearch.client;
+
+import io.redisearch.*;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.Protocol;
+import redis.clients.jedis.commands.ProtocolCommand;
+
+import org.json.JSONObject;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ClientJsonTest {
+
+    public static final String JSON_ROOT = "$";
+
+    private enum JsonCommands implements ProtocolCommand {
+
+        SET;
+
+        @Override
+        public byte[] getRaw() {
+            return ("JSON." + name()).getBytes();
+        }
+    }
+
+    private static final String INDEX_NAME = "json-index";
+
+    private static Client search;
```

Comment:
@sazzad16 why is it static?

Reply:
@gkorland For doing search.close() in tearDown().